docs/source/workflow/generate-types.mdx

```diff
@@ -13,56 +13,47 @@ You can use these generated TS types in your resolvers to type-check that your r
 
 ## Setting up your project
 
-We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/docs/config-reference/schema-field) to provide a schema to GraphQL Code Generator. Below, we'll show the most common method, which requires our schema to be in a `.graphql` file.
+We'll use the [GraphQL Code Generator](https://www.the-guild.dev/graphql/codegen) library to generate types based on our GraphQL schema. There are [multiple ways](https://www.the-guild.dev/graphql/codegen/plugins/typescript/typescript-resolvers#usage-example) to provide a schema to GraphQL Code Generator. Below, we'll show one of the common methods, beginning by wrapping our schema in a `gql` template literal tag from the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) package.
 
-If you haven't already, move your server's schema into a `.graphql` file, like so:
+If you haven't already, begin by importing the `graphql-tag` library:
 
-```graphql title="src/schema.graphql"
-type Query {
-  books: [Book]
-}
+```bash
+npm install graphql-tag
+```
 
-type Book {
-  title: String
-  author: String
-}
+The `gql` template literal tag from `graphql-tag` can parse a Schema Definition Language (SDL) string into an abstract syntax tree (AST). Under the hood, Apollo Server handles this for you when you provide your schema as a string. 
```

Comment:
> Under the hood, Apollo Server handles this for you when you provide your schema as a string.

This is correct but maybe said in a confusing way that doesn't apply to this guide. In this guide, we do _not_ pass the schema as a string so the statement doesn't really pertain. I might adjust the sentence to say that AS will gladly accept the schema either as a string or AST.

Reply:
I like that! How does this feel:

> The `gql` template literal tag from `graphql-tag` can parse a Schema Definition Language (SDL) string into an abstract syntax tree (AST). Apollo Server accepts a string or an AST as a valid option for defining a schema.

Reply:
Great!